plan.md

```diff
@@ -4,4 +4,6 @@ Write plans and concepts
   - Simplify many Behaviors to simply wrap one action
   - We should be able to ask Behaviors if they can fulfil a requirement (a need, inventory status, environmental status)
   - Behaviors can then recursively use each other to create more complex behavior
-  
\ No newline at end of file
+  
+  
+- Have chat
```

Comment:
We could also make use of the wiki functionality github provides to keep track of current functionality/planned features.